less/pageLevelProgressIndicator.less

```diff
@@ -0,0 +1,93 @@
+.menu-title-inner,
+.menu-item-title-inner,
+.page-title-inner,
+.article-title-inner,
+.block-title-inner,
+.component-title-inner {
+    display: inline-block;
+}
+
+.pagelevelprogress-indicator {
+    display: inline-block;
+    box-sizing: border-box;
+    height: 12px;
+    width: 24px;
+    border: 2px solid @item-color-hover;
+    border-radius: 50px;
+    background-color: @foreground-color-inverted;
+    vertical-align: middle;
+    overflow: hidden;
+
+    &.complete .pagelevelprogress-indicator-inner {
+        width: 100%;
+    }
+}
+
+.pagelevelprogress-indicator-inner {
+    position: relative;
+    overflow: hidden;
+    box-sizing: border-box;
+    height: 8px;
+    border: 1px solid @foreground-color-inverted;
+    border-radius: 25px;
+}
+
+.pagelevelprogress-indicator-bar {
+    position: absolute;
+    min-width: 4px;
```

Comment:
The min-width feels wrong to me since the styling makes it obvious it's fillable now and being partially filled makes me think I've already seen the page.

Strictly a personal preference, mind, _you can ignore._

Reply:
I do see your point. 

With the new styling I like that the initial state provides a total description of what is expected - there is a container, with empty space and material to fill it, but it is as yet, not full.

In the original styling we had a container and a potentially empty space, but no indication that there would be material to fill it or whether the material was already present.

I'd worry slightly, that without the presence of the filling material on the initial viewing, that the issue of an unknowably empty space will reappear. This is especially apparent when opposing colours are chosen in a course, such as the example course which has a dark drawer, notify or header colour. Apposing colours for this same UI element allow for a situation where no distinct difference can be discerned between the material colour and the empty space.

For the moment, I quite like Matt's suggestion of having a little material shown initially, based on the above theory which is admittedly, entirely conjecture.

Reply:
![image](https://user-images.githubusercontent.com/7974663/45693348-cd407280-bb54-11e8-9851-70a2c686c870.png)

^ they are squared off now to make them look less toggley

Reply:
Let's humbly agree to disagree.